src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -1632,8 +1562,6 @@ class device {
         for(int32_t i = 0; i < map->m_count; i++) {
             if(map->m_mem_data[i].m_type == MEM_STREAMING)
                 continue;
-            if(!strncmp((const char*)map->m_mem_data[i].m_tag, "HOST", 4))
```

Comment:
This looks like  a merging issue.  Please revert this change

Reply:
Fixed this issue. Thanks.